test-suite/Makefile

```diff
@@ -38,9 +38,9 @@ include ../Makefile.common
 BIN := $(shell cd ..; pwd)/bin/
 COQFLAGS?=
 
+COQLIB?=
 ifeq ($(COQLIB),)
-  # This method of setting `pwd` won't work on win32 OCaml
-  COQLIB := $(shell cd ..; pwd)
+  COQLIB := $(shell ocaml ocaml_pwd.ml ..)
```

Comment:
It looks like ocaml_pwd is quoting the filename, then the quotes survive throught the environment and Coq doesn't unquote back so it tries to read `'/bla/'theories`

Reply:
Previously it was used only by dune which uses it through `make ... COQLIB=%{read:libpath.inc}` so the quotes get removed by the shell.

Reply:
Why do we even quote in ocaml_pwd, shouldn't the caller be in charge of quoting as they need? @ejgallego

Reply:
That's tricky business as far as I remember, the windows filename quoting rules are special so you cannot quote in bash for example AFAICT.

Reply:
Adding the `echo` does the trick.  Unless one of you has a better fix (e.g. to ocaml_pwd) that you can make in, say, the next week, let's merge this fix.

Reply:
Maybe we could just require the caller of the `Makefile` to set `COQLIB` else error?

Anyways, I am OK with this fix, thanks @jfehrle

Reply:
I would avoid changing the requirements for running the test suite unless there's a strong reason to do so.  Less disruption to others.

@gares, would you review and merge if this is agreeable?